scripts/apps/authoring/authoring/services/AutosaveService.ts

```diff
@@ -60,33 +60,44 @@ export class AutosaveService {
     /**
      * Auto-saves an item
      */
-    save(item: IArticle, orig: IArticle, timeout: number = AUTOSAVE_TIMEOUT, callback) {
+    save(
+        item: IArticle,
+        orig: IArticle,
+        timeout: number = AUTOSAVE_TIMEOUT,
+        callback, applyAsync?: () => Promise<void>,
```

Comment:
two arguments on same line? ;D

wouldn't it work to use a callback so you don't need to pass applyAsync argument?